open-craft/ecommerce
# [BB-2837] Pulled latest Arabic translations from transifex

This PR adds the latest Arabic translations from transifex.

**JIRA tickets**: https://tasks.opencraft.com/browse/BB-2837

~~**Discussions**:~~
~~**Dependencies**:~~

**Screenshots**: 
![localhost_18130_basket_](https://user-images.githubusercontent.com/1010244/108539454-3a4e2880-730a-11eb-8b69-a0d24e626300.png)

~~**Sandbox URL**:~~

**Merge deadline**: ASAP

**Testing instructions**:

1. Pull this PR into your local devstack.
2. Switch language to Arabic in LMS.
3. Check e-commerce UI has up-to-date translations.

**Author notes and concerns**:
N/A

**Reviewers**
